zerver/actions/user_groups.py

```diff
@@ -169,11 +171,24 @@ def do_send_create_user_group_event(
     members: list[UserProfile],
     direct_subgroups: Sequence[UserGroup] = [],
 ) -> None:
+    if user_group.creator is not None:
+        creator_id = user_group.creator.id
+    else:
+        creator_id = None
+
+    # This check is necessary to make mypy happy
+    date_created = (
```

Comment:
The need for this check arises from the fact that the `datetime_to_timestamp` function only accepts `datetime` objects as input, while the `date_created` field can potentially contain `None` values.